packages/validator-ajv8/src/validator.ts

```diff
@@ -199,20 +202,67 @@ export default class AJV8Validator<
    * At some point, components should be updated to support ajv.
    *
    * @param errors - The list of AJV errors to convert to `RJSFValidationErrors`
-   * @private
+   * @protected
    */
-  private transformRJSFValidationErrors(
-    errors: ErrorObject[] = []
+  protected transformRJSFValidationErrors(
+    errors: ErrorObject[] = [],
+    uiSchema?: UiSchema<T, S, F>
   ): RJSFValidationError[] {
     return errors.map((e: ErrorObject) => {
-      const { instancePath, keyword, message, params, schemaPath } = e;
+      const {
+        instancePath,
+        keyword,
+        params,
+        schemaPath,
+        parentSchema,
+        ...rest
+      } = e;
+      let { message = "" } = rest;
+
       let property = instancePath.replace(/\//g, ".");
+
       let stack = `${property} ${message}`.trim();
+
       if ("missingProperty" in params) {
         property = property
           ? `${property}.${params.missingProperty}`
           : params.missingProperty;
-        stack = message!;
+
+        const currentProperty: string = params.missingProperty;
+
+        const uiSchemaTitle = getUiOptions(
```

Comment:
Let's keep the variables together
```suggestion
        const currentProperty: string = params.missingProperty;
        const uiSchemaTitle = getUiOptions(
```